src/services/localeStorage/index.ts

```diff
@@ -46,6 +51,36 @@ interface LocalStorageSchema {
   lastSdkCache: { timestamp: number; ttl: number };
 }
 
-export const lsService = new ManagedLocalStorage<LocalStorageSchema>((key) =>
-  [APP_ID, APP_VERSION, key].join('-')
+// ************************** /
+// LOCAL STORAGE MIGRATION AND CLEAN_UP
+// ************************** /
+
+/**
+ * An array of migration objects, each designed to handle specific versions of stored items.
```

Comment:
emphasize that order matters and that each migration step is applied after the other, so that even if someone open the app after a year and 20 upgrades, there will be a series of migration steps that take him all the way to now